src/protagonist/API.Tests/Integration/ModifyAssetTests.cs

```diff
@@ -1073,14 +1135,80 @@ public async Task Put_Existing_Asset_Returns400_IfDeliveryChannelsNull()
             ""mediaType"": ""image/tiff""
         }}";
         
+        A.CallTo(() =>
+                EngineClient.SynchronousIngest(
+                    A<Asset>.That.Matches(r => r.Id == assetId),
+                    A<CancellationToken>._))
+            .Returns(HttpStatusCode.OK);
+        
+        // act
+        var content = new StringContent(hydraImageBody, Encoding.UTF8, "application/json");
+        var response = await httpClient.AsCustomer(99).PutAsync(assetId.ToApiResourcePath(), content);
+        
+        // assert
+        response.StatusCode.Should().Be(HttpStatusCode.OK);
+        
+        var asset = dbContext.Images.Include(i => i.ImageDeliveryChannels).Single(x => x.Id == assetId);
+        asset.Id.Should().Be(assetId);
+        asset.MaxUnauthorised.Should().Be(-1);
+        asset.ImageDeliveryChannels.Should().HaveCount(3).And.Contain(i =>
```

Comment:
This could be simplified to `asset.ImageDeliveryChannels.Should().BeEquivalentTo(deliveryChannels);`